discovery-provider/docker-compose.pedalboard.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.9"
```

Comment:
I have some (perhaps annoying) opinions that we should name this docker-compose.plugins.yml because this repo gets pretty public facing, and i think for the understandability quotient "plugin" is more readable.

Maybe the distinction is that pedalboard is the framework to build plugins, but what actually is deployed are plugins?

Wdyt @alecsavvy @sabrina-kiam

Reply:
agree